README.md

```diff
@@ -1,20 +1,43 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# React-Redux Setup
+# Nemwel ZOO
```

Comment:
- [ ] Currently, you have only one unit test for the Zoo component. In your app, kindly add more unit tests for some of your components.

- [ ] **Kindly ensure all implemented changes are pushed to the live demo.**